# **State the Hypotheses statement**

Null Hypothesis (H0): There is no significant difference between the two groups.

Alternative Hypothesis (H1): There is a significant difference between the two groups

# **Calculate the Test Statistic**

In [1]:
x_bar = 3050
mu = 4000
n = 25
sigma = 125

In [3]:
import math
Zclac = (x_bar-mu)/(sigma/math.sqrt(n)) # Test statistic formula
print("test statistic value :",Zclac)

test statistic value : -38.0


# **Determine the Critical Value**

In [5]:
from scipy.stats import norm
import numpy as np
alph = 0.05
Z_critical = norm.ppf(1-0.05)  # ppf is  Percent-Point Function
Z_cr = np.round(Z_critical,3)  # taken only after three decimals
print("Critical Z_value :",Z_cr)

Critical Z_value : 1.645


the critical Z-value for a one-tailed test at a 5% significance level (α = 0.05) is approximately 1.645.

# **Make a Decision**

The test statistic we calculated is -38.

The critical value for a one-tailed test at alph = 0.05 critical z_value = 1.645.

Since the test statistic -38 is much less than 1.645 (-38<1.645), it is far in the left tail of the normal distribution.

The test statistic (-38) is less than the critical value (1.645), so we fail to reject the null hypothesis.

Ho is rejected and H1 is accepted

# **Conclusion**

There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than what the cost model suggests.

The null hypothesis stated that the average weekly operating cost is 4,000 Rs, as per the cost model

The alternative hypothesis suggested that the average cost was greater than 4,000 Rs.

However, the test statistic (-38) was much smaller than the critical value (1.645), indicating that the sample mean of 3,050 Rs is significantly lower than the theoretical value of 4,000 Rs.

Therefore, we fail to reject the null hypothesis, and there is insufficient statistical evidence to conclude that the operating costs are higher than the theoretical value of 4,000 Rs. The evidence does not support the restaurant owners' claim of increased operating costs
